src/main/java/com/intel/gkl/pairhmm/IntelPairHmm_omp.java

```diff
@@ -0,0 +1,92 @@
+package com.intel.gkl.pairhmm;
+
+import com.intel.gkl.IntelGKLUtils;
+import org.broadinstitute.gatk.nativebindings.pairhmm.HaplotypeDataHolder;
+import org.broadinstitute.gatk.nativebindings.pairhmm.PairHMMNativeArguments;
+import org.broadinstitute.gatk.nativebindings.pairhmm.PairHMMNativeBinding;
+import org.broadinstitute.gatk.nativebindings.pairhmm.ReadDataHolder;
+
+import java.io.File;
+
+/**
+ * Created by pnvaidya on 10/12/16.
+ */
+public class IntelPairHmm_omp implements PairHMMNativeBinding {
```

Comment:
Can we derive `IntelPairHmm_omp` from `IntelPairHmm` so we reduce the amount of duplicate code?